src/components/GrowerDetail.js

```diff
@@ -19,6 +19,8 @@ import { GrowerContext } from '../context/GrowerContext';
 import EditGrower from './EditGrower';
 import OptimizedImage from './OptimizedImage';
 import LinkToWebmap from './common/LinkToWebmap';
+import { CopyButton } from './CaptureDetailDialog';
+import CopyNotification from './CopyNotification';
```

Comment:
Move this to `src/components/common/` as it's a component shared across other components.